### DISCUSSION TITLE
Provided key is corrupted or contains invalid data login the wallet

### DISCUSSION THREAD

**Discussion by Yavuz134** [IBAX-io/go-ibax#41] [Bug Submission]
![Uploading 1642090457746944776931968338314.jpg…]()

**Comment by emre2587** [comment]
![Ekran görüntüsü 2022-01-13 201814](https://user-images.githubusercontent.com/88062171/149378078-977962b1-9d4a-4cb1-8fcb-1b447c06782b.png)

aynı sorun bende de var.

**Reply by HalloKyle** [reply]
> ![Ekran görüntüsü 2022-01-13 201814](https://user-images.githubusercontent.com/88062171/149378078-977962b1-9d4a-4cb1-8fcb-1b447c06782b.png)
> 
> aynı sorun bende de var.

https://github.com/IBAX-io/go-ibax/discussions/62

**Comment by HalloKyle** [comment]
https://github.com/IBAX-io/go-ibax/discussions/62

**Comment by HalloKyle** [comment]
please check this post for solution

**Comment by Yavuz134** [comment]
necessary information telegram dav given thanks

Galaxy cihazımdan gönderildi



-------- Orijinal mesaj --------
Kimden: Kyle ***@***.***>
Tarih: 22.01.2022 13:26 (GMT+03:00)
Alıcı: IBAX-io/go-ibax ***@***.***>
Cc: Yavuz134 ***@***.***>, Author ***@***.***>
Konu: Re: [IBAX-io/go-ibax] Provided key is corrupted or contains invalid data login the wallet (Discussion #41)


[Ekran görüntüsü 2022-01-13 201814]<https://user-images.githubusercontent.com/88062171/149378078-977962b1-9d4a-4cb1-8fcb-1b447c06782b.png>

aynı sorun bende de var.

#62<https://github.com/IBAX-io/go-ibax/discussions/62>

—
Reply to this email directly, view it on GitHub<https://github.com/IBAX-io/go-ibax/discussions/41#discussioncomment-2021947>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AXJJ6TQQFBKHKNNMZU6I4BDUXKA5LANCNFSM5L4JCCWQ>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
